includes/class-wc-stripe-intent-controller.php

```diff
@@ -326,14 +327,105 @@ public function create_payment_intent( $order_id = null ) {
 		];
 	}
 
+	/**
+	 * Handle AJAX request for updating a payment intent for Stripe UPE.
+	 *
+	 * @since x.x.x
+	 *
+	 * @throws Exception - If nonce or setup intent is invalid.
```

Comment:
This can be removed, no exception is thrown, a valid JSON is always returned

Reply:
Nice catch! Got them both removed.